src/tic_tac_clojure/validators.clj

```diff
@@ -0,0 +1,18 @@
+(ns tic-tac-clojure.validators
+  (:gen-class)
+  (:require [tic-tac-clojure.utils :refer :all]))
+
+(defn is-number?
+  [str]
+  (do
+    (try (convert-to-num str) 
+          true
+          (catch Exception e false))))
+  
+(defn in-range?
+  [start end val]
+  (and (>= val start) (<= val end)))
+
+(defn valid-numeric-selection? 
```

Comment:
I couldn't find where you call this function; it appears to be dead code.

Reply:
You're right. I replaced that function with `get-valid-num-input` in the cli namespace.